package.json

```diff
@@ -0,0 +1,208 @@
+{
+  "name": "cruise-ships",
+  "version": "1.0.0",
+  "description": "Cruise Ships 🚢 Javascript Object-Oriented code for a Cruise Ships. Building and working with many objects that allow a cruise ship to operate. Running the app To run clone to local machine.",
+  "main": "index.js",
+  "dependencies": {
+    "ansi-regex": "^5.0.1",
```

Comment:
Are all these dependencies actually needed? There seems to be an awful lot here for such a small app